src/jit/gentree.h

```diff
@@ -574,11 +574,21 @@ struct GenTree
         return isContained() && IsCnsIntOrI() && !isUsedFromSpillTemp();
     }
 
+#if FEATURE_X87_DOUBLES
```

Comment:
>FEATURE_X87_DOUBLES [](start = 4, length = 19)

Note that FEATURE_X87_DOUBLES is meant for tracking floating point code specific to x87 which only legacy x86 supports.

Using that for float vs dbl constants doesn't seem right to me.